cla_public/static-templates/errors/50x.html

```diff
@@ -121,51 +121,51 @@
         <div class="govuk-phase-banner">
           <p class="govuk-phase-banner__content">
             <span class="govuk-phase-banner__text" id="aria-feedback">
-              <a href="{{ url_for('base.reasons_for_contacting')}}">Report a problem</a> if you need help or would like to give feedback to improve this service
+              <a href="{{ url_for('base.reasons_for_contacting')}}">{% trans %}Report a problem</a> if you need help or would like to give feedback to improve this service{% endtrans %}
             </span>
           </p>
         </div>
         <main class="govuk-main-wrapper " id="main-content" role="main">
-          <h1 class="govuk-heading-xl">This service is temporarily unavailable</h1>
-          <p class="govuk-body">This service is not available right now, but we're working hard to get things up and running again.</p>
-          <p class="govuk-body">You can try again later or contact us. </p>
-          <p class="govuk-body">Telephone:<br />
-            <strong class="govuk-!-font-weight-bold">0345 345 4345</strong>
+          <h1 class="govuk-heading-xl">{% trans %}This service is temporarily unavailable{% endtrans %}</h1>
+          <p class="govuk-body">{% trans %}This service is not available right now, but we're working hard to get things up and running again.{% endtrans %}</p>
+          <p class="govuk-body">{% trans %}You can try again later or contact us.{% endtrans %}</p>
+          <p class="govuk-body">{% trans %}Telephone:<br/>
+            <strong class="govuk-!-font-weight-bold">0345 345 4345{% endtrans %}</strong>
```

Comment:
Again, I'd omit the HTML and telephone number from the translation - the only translatable element here is telephone.  
`<p class="govuk-body">{% trans %}Telephone{% endtrans %}:<br/>
            <strong class="govuk-!-font-weight-bold">0345 345 4345</strong>`